tests/unit/test_noise_form.py

```diff
@@ -150,6 +150,8 @@ def test_noise_order(mocker, dummy_data, dummy_config_noise_numbers):
     # FIXME: would be better to mock the dataset instead of using census
     noise_dataset(DATASETS.census, dummy_data, dummy_config_noise_numbers, 0)
 
+    # This is getting the string of each noise type. It is filtering down to string type for each noise type
+    # while not including mock calls that also contain that string.
```

Comment:
But why do we care if this is a string? You should communicate these things:
* What is `x[0]`?
* What is `x[1][0]`? (the first argument of the call)
* Why do we care if the first argument of the mock call is a string?
* Mention the duplicate calls occurring due to having two methods mocked on the noise type*